src/js/app.js

```diff
@@ -30,7 +37,6 @@ indexApp.controller('indexCtrl', ($scope) => {
     const fileLength = $scope.myInvertedIndex.files[fileName].length;
     return Array.from({ length: fileLength }, (value, key) => key);
   };
-
   $scope.indexFile = () => {
     $scope.myInvertedIndex.createIndex($scope.currentFileName);
```

Comment:
I feel there is not need to attach $scope to myInvertedIndex since you are not using it in your view